analytics_data_api/v0/serializers.py

```diff
@@ -49,6 +49,26 @@ class ProblemSubmissionCountSerializer(serializers.Serializer):
     correct = serializers.IntegerField(default=0)
 
 
+class ConsolidatedAnswerDistributionSerializer(serializers.Serializer):
```

Comment:
You *may* be able to get away with subclassing `ProblemResponseAnswerDistributionSerializer` and just adding the `consolidated_variant` field.